src/Services/EncounterService.php

```diff
@@ -242,6 +256,69 @@ public function getEncounterForPatient($pid, $eid)
         return sqlQuery($sql, array($pid, $eid));
     }
 
+    public function insertEncounter($pid, $data)
+    {
+
+        $encounter = generate_id();
+        
```

Comment:
reduce these 2 empty lines to one empty line

Reply:
Done